### ISSUE TITLE
triplet loss function

### ISSUE THREAD

**Issue by hpdsy-zx** [peblair/jrc-names-retrieval#1]
Hi, I read the paper recently and have some questions, could you please help with these questions ? First is about triplet loss function, I first made n an and n^+ for training, using L {contrast}. After training, I made a model and trained a data epoch. Then I selected negative data according to the method introduced in the paper, and conducted negative sample training of L {contrast}. After training the epoch, I build the triplet of the triplet and train it with L {triplet}. Is that right? Second is What is the m selection in contrast loss and triplet loss in the article? Thanks!
